app/components/header/header.component.html

```diff
@@ -82,88 +82,25 @@
           <span class="toolbar-Text">Stash</span>
         </li>
       </ul>
-
-      <div class="collapse nav navbar-nav nav-collapse" id="nav-collapse1">
-        <form class="navbar-form navbar-right form-inline" role="form" style="width:280px;">
-          <div class="form-group">
-            <label class="sr-only" for="Email">User name</label>
-            <img src="./assets/error.png" class="error-icon" id="username-error-icon">
-            <input type="text" class="form-control" id="Email1" placeholder="Username or Email" autofocus required
-              style="width: 206px !important" />
+      <ul class="navbar-nav navbar-right hidden-xs">
+        <li>
+          <label id="githubname" style="color:white"></label>
```

Comment:
Should not use inline styles

Reply:
Will make new issue for this because other html files use inline style too